sentry/sentry.conf.example.py

```diff
@@ -220,3 +220,4 @@
 
 # BITBUCKET_CONSUMER_KEY = 'YOUR_BITBUCKET_CONSUMER_KEY'
 # BITBUCKET_CONSUMER_SECRET = 'YOUR_BITBUCKET_CONSUMER_SECRET'
+
```

Comment:
Revert please?

Reply:
Added a space because the script inserts a new line at the end of the file with the known Relay keys (and it looks bad if it comes immediately after the comment).

Reply:
Okay